src/services/cleeng.account.service.ts

```diff
@@ -31,8 +31,27 @@ import type {
   ChangePasswordWithOldPassword,
   UpdatePersonalShelves,
 } from '#types/account';
+import cleengAuthService from '#src/services/cleeng.auth.service';
 
-export const setEnvironment = () => true;
+const getCustomerIdFromAuthData = (auth: AuthData) => {
+  const decodedToken: JwtDetails = jwtDecode(auth.jwt);
+  return decodedToken.customerId;
+};
+
+export const initialize = async (config: Config, logoutCallback: () => Promise<void>) => {
+  await cleengAuthService.initialize(!!config.integrations.cleeng?.useSandbox, logoutCallback);
+};
+
+export const getAuthData = async () => {
+  if (cleengAuthService.tokens) {
+    return {
+      jwt: cleengAuthService.tokens.accessToken,
+      refreshToken: cleengAuthService.tokens.refreshToken,
+    } as AuthData;
```

Comment:
We can remove `customerToken` from the AuthData since it not being used anymore

Reply:
Sure!

Reply:
And now you don't have to cast the object to AuthData anymore